packages/utilities/psammead-locales/moment/index.stories.jsx

```diff
@@ -79,16 +79,19 @@ locales.forEach(({ name, locale }) => {
     () => (
       <Fragment>
         <Table>
-          <tr>
-            <th>British English</th>
-            <th>{name}</th>
-          </tr>
-          {funcs.map(func => (
+          <tbody>
             <tr>
-              <td>{func('en-gb')}</td>
-              <td>{func(locale)}</td>
+              <th>British English</th>
+              <th>{name}</th>
             </tr>
-          ))}
+            {funcs.map((func, index) => (
+              /* eslint-disable react/no-array-index-key */
+              <tr key={index}>
```

Comment:
Figured it would be ok to use `index` as a key here - as opposed to generating unique ids, because even though the shown value is computed, the func array isn't altered

Reply:
You could use `func('en-gb')` instead? But I think you're right; it won't have any major effect here